### ISSUE TITLE
LottieDrawable.setAlpha does not make effect

### ISSUE THREAD

**Issue by leehong2005** [airbnb/lottie-android#419]
I encountered a problem, I set lottie drawable alpha, but no effect. My code is following:

```
Drawable drawable = animationView.getDrawable();
        if (drawable instanceof LottieDrawable) {
            int alpha = (int) (255 * ((100 - progress) / 100f));
            ((LottieDrawable) drawable).setAlpha(alpha);
            animationView.invalidate();
        }
```

What should I do?

**Comment by leehong2005** [comment]
SolidLayer#draw() method code is as following:

```
@Override public void drawLayer(Canvas canvas, Matrix parentMatrix, int parentAlpha) {
    int backgroundAlpha = Color.alpha(layerModel.getSolidColor());
    if (backgroundAlpha == 0) {
      return;
    }

    int alpha = (int) ((backgroundAlpha / 255f * transform.getOpacity().getValue() / 100f) * 255);
    paint.setAlpha(alpha);
    if (alpha > 0) {
      updateRect(parentMatrix);
      canvas.drawRect(rect, paint);
    }
  }
```

In this method, does NOT use the parentAlpha， I think the code should add the parent alpha, 

```
 int alpha = (int) ((backgroundAlpha / 255f * transform.getOpacity().getValue() / 100f) * 255); 
```
should be changed to ：

```
int alpha = (int) (parentAlpha / 255f * (backgroundAlpha / 255f * transform.getOpacity().getValue() / 100f) * 255);
```

Is it right?

**Comment by gpeal** [comment]
@leehong2005 Yes, this seems correct to me. Want to put up a PR?

**Comment by gpeal** [comment]
This is fixed in the latest release (2.2.2)
